apps/admin-x-activitypub/src/views/Feed/Note.tsx

```
@@ -0,0 +1,205 @@
+                                            <div className='flex w-full'>
+                                                <span className='text-gray-700 after:mx-1 after:font-normal after:text-gray-700 after:content-["·"]'>{getUsername(post.actor)}</span>
+                                                <span className='text-gray-700'>{renderTimestamp(object, !object.authored)}</span>
+                                            </div>
+                                        </div>
+                                    </div>
+                                        }
+
+                                        {threadParents.map((item) => {
+                                            return (
+                                                item.object.type === 'Tombstone' ? (
+                                                    <DeletedFeedItem last={false} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)